lib/podcast_importer.rb

```diff
@@ -0,0 +1,267 @@
+# import script for a podcast live in feeder & dovetail
+require 'prx_access'
+require 'addressable/uri'
+require 'feedjira'
+require 'itunes_category_validator'
+
+class PodcastImporter
```

Comment:
That, and (per my other comment) when would these lib importers be used vs the model importer?